app/experimenter/nimbus-ui/src/components/PageEditBranches/index.tsx

```diff
@@ -36,6 +40,20 @@ const PageEditBranches: React.FunctionComponent<PageEditBranchesProps> = () => {
           }}
         />
         <h2 className="mt-3 h4">Branches</h2>
+        <p>
+          If you want, you can add a <strong>feature flag</strong> configuration
+          to each branch. Experiments can only change one flag at a time.{" "}
+          <a href={BRANCHES_DOC_URL}>Learn more</a>
```

Comment:
This might need to be a `<LinkExternal>` so we're not accidentally taking folks out of the flow.

Reply:
Switched to LinkExternal